mandatory/4-tax.js

```diff
@@ -5,7 +5,10 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
+  return price * 0.2 + price;
```

Comment:
Well done with these functions. A minor improvement to this code would be:

```suggestion
  return price * 1.2;
```